### PR TITLE
Update to Aspire 8.0 GA [WIP]

### PR DESCRIPTION
**Pull request by adityamandaleeka**

### CODE REVIEW

File: nuget.config [hunk 1]
```diff
@@ -4,11 +4,22 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="nuget" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
```
Comment (adityamandaleeka) [inline nuget.config:7]:
```suggestion
    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-aspire-d215c528/nuget/v3/index.json" />
```

File: nuget.config [hunk 2]
```diff
@@ -4,11 +4,27 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="nuget" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet-tools-pub" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-aspire-d215c528/nuget/v3/index.json" />
+    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
```
Comment (joperezr) [inline nuget.config:8]: Why do you need this? I would have expected for you to only need the one above.

File: nuget.config [hunk 3]
```diff
@@ -4,11 +4,27 @@
...
   </packageSources>
 
   <packageSourceMapping>
     <!-- key value for <packageSource> should match key values from <packageSources> element -->
     <clear />
+    <packageSource key="dotnet-tools-pub">
+      <package pattern="Aspire.*" />
+      <package pattern="Microsoft.Extensions.ServiceDiscovery*" />
+    </packageSource>
+    <packageSource key="dotnet-tools">
```
Comment (joperezr) [inline nuget.config:19]: I think this feed should be able to go.

File: Directory.Packages.props [hunk 4]
```diff
@@ -5,7 +5,8 @@
     <AspnetVersion>8.0.4</AspnetVersion>
     <MicrosoftExtensionsVersion>8.4.0</MicrosoftExtensionsVersion>
     <EfVersion>8.0.4</EfVersion>
-    <AspireVersion>8.0.0-preview.7.24251.11</AspireVersion>
+    <AspireVersion>8.0.0</AspireVersion>
+    <AspirePreviewVersion>8.0.0-preview.8.24258.2</AspirePreviewVersion>
```
Comment (joperezr) [inline Directory.Packages.props:9]: NIT: Probably a more accurate property name would be something like `AspireUnstablePackagesVersion` or something along those lines.

**Review by joperezr — APPROVED** [review]

File: Directory.Packages.props [hunk 5]
```diff
@@ -21,7 +22,7 @@
     <PackageVersion Include="Aspire.Npgsql.EntityFrameworkCore.PostgreSQL" Version="$(AspireVersion)" />
     <PackageVersion Include="Aspire.RabbitMQ.Client" Version="$(AspireVersion)" />
     <PackageVersion Include="Aspire.StackExchange.Redis" Version="$(AspireVersion)" />
-    <PackageVersion Include="Aspire.Azure.AI.OpenAI" Version="$(AspireVersion)" />
+    <PackageVersion Include="Aspire.Azure.AI.OpenAI" Version="$(AspireUnstablePackagesVersion)" />
```
Comment (captainsafia) [inline Directory.Packages.props:25]: What's the reason we need a separate package version for this?

Comment (adityamandaleeka) [reply Directory.Packages.props:25]: Some packages aren't GA-ing yet

**Review by danmoseley — APPROVED** [review]

**Comment by joperezr** [comment]
This is now ready to go. Ok to merge @adityamandaleeka ?

**Pull request merged**